app/uk/gov/hmrc/agentclientrelationships/services/FindRelationshipsService.scala

```diff
@@ -45,6 +46,16 @@ class FindRelationshipsService @Inject()(des: DesConnector, val metrics: Metrics
     clientId: Vrn)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[ActiveRelationship]] =
     des.getActiveClientRelationships(clientId)
 
+  def getActiveRelationships(taxIdentifier: TaxIdentifier)(
+    implicit hc: HeaderCarrier,
+    ec: ExecutionContext): Future[Option[ActiveRelationship]] =
+    taxIdentifier match {
+      case MtdItId(_) | Vrn(_) | Utr(_) => des.getActiveClientRelationships(taxIdentifier)
+      case e =>
+        Logger(getClass).warn(s"No Active Relationship found. Unsupported Identifier ${e.getClass.getSimpleName}")
```

Comment:
we can remove the wording `No Active Relationship found.` because its not actually looking for any relations in this case

Reply:
ok